--- a/Restoration/12_UnityReskin_20260922/Assets/Prefabs/Runtime/RecoveredMain.prefab
+++ b/Restoration/12_UnityReskin_20260922/Assets/Prefabs/Runtime/RecoveredMain.prefab
@@ -15891,8 +15891,8 @@
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_AnchoredPosition: {x: 4.0820866, y: -5.3630285}
+  m_SizeDelta: {x: 343.71146, y: 136.9635}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1175646791480262997
 MonoBehaviour:
@@ -15981,7 +15981,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -16238,7 +16238,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1175646791480263007
 MonoBehaviour:
@@ -16326,7 +16326,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -16416,7 +16416,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1175646791480263000
 MonoBehaviour:
@@ -16505,7 +16505,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -16592,10 +16592,10 @@
   m_Children: []
   m_Father: {fileID: 1175646791480262985}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -7.2701845, y: 10.145175}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1824845157082445496
 MonoBehaviour:
@@ -16803,10 +16803,10 @@
   m_Children: []
   m_Father: {fileID: 1175646791480262989}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &8181515030794109160
 MonoBehaviour:
@@ -17015,10 +17015,10 @@
   m_Children: []
   m_Father: {fileID: 1175646791480262991}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &6374444575512743863
 MonoBehaviour:
@@ -22805,8 +22805,8 @@
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_AnchoredPosition: {x: 4.0820866, y: -5.3630285}
+  m_SizeDelta: {x: 343.71146, y: 136.9635}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1543360988313136277
 MonoBehaviour:
@@ -22895,7 +22895,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -23152,7 +23152,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1543360988313136287
 MonoBehaviour:
@@ -23240,7 +23240,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -23330,7 +23330,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1543360988313136280
 MonoBehaviour:
@@ -23419,7 +23419,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -23506,10 +23506,10 @@
   m_Children: []
   m_Father: {fileID: 1543360988313136265}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -7.2701845, y: 10.145175}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1148365902302515465
 MonoBehaviour:
@@ -23717,10 +23717,10 @@
   m_Children: []
   m_Father: {fileID: 1543360988313136269}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &8336459692023299411
 MonoBehaviour:
@@ -23929,10 +23929,10 @@
   m_Children: []
   m_Father: {fileID: 1543360988313136271}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &6182350930745565989
 MonoBehaviour:
@@ -39912,8 +39912,8 @@
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_AnchoredPosition: {x: 4.0820866, y: -5.3630285}
+  m_SizeDelta: {x: 343.71146, y: 136.9635}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &2730608590170936925
 MonoBehaviour:
@@ -40002,7 +40002,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -40259,7 +40259,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &2730608590170936919
 MonoBehaviour:
@@ -40347,7 +40347,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -40437,7 +40437,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &2730608590170936912
 MonoBehaviour:
@@ -40526,7 +40526,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -40613,10 +40613,10 @@
   m_Children: []
   m_Father: {fileID: 2730608590170936897}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -7.2701845, y: 10.145175}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &5408132952394150896
 MonoBehaviour:
@@ -40824,10 +40824,10 @@
   m_Children: []
   m_Father: {fileID: 2730608590170936901}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &6327638101108633749
 MonoBehaviour:
@@ -41036,10 +41036,10 @@
   m_Children: []
   m_Father: {fileID: 2730608590170936903}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1172889475544162374
 MonoBehaviour:
@@ -45299,10 +45299,10 @@
   m_Children: []
   m_Father: {fileID: 3262282531854528311}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &6263817822924323253
 MonoBehaviour:
@@ -45511,10 +45511,10 @@
   m_Children: []
   m_Father: {fileID: 3262282531854528307}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -7.2701845, y: 10.145175}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &2387744081387396777
 MonoBehaviour:
@@ -45722,10 +45722,10 @@
   m_Children: []
   m_Father: {fileID: 3262282531854528309}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &7177864758418968480
 MonoBehaviour:
@@ -46102,8 +46102,8 @@
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_AnchoredPosition: {x: 4.0820866, y: -5.3630285}
+  m_SizeDelta: {x: 343.71146, y: 136.9635}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &3262282531854528303
 MonoBehaviour:
@@ -46192,7 +46192,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -46282,7 +46282,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &3262282531854528290
 MonoBehaviour:
@@ -46371,7 +46371,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -46461,7 +46461,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &3262282531854528293
 MonoBehaviour:
@@ -46549,7 +46549,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -62129,10 +62129,10 @@
   m_Children: []
   m_Father: {fileID: 4368578466105820056}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -7.2701845, y: 10.145175}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1057078821336938133
 MonoBehaviour:
@@ -62340,10 +62340,10 @@
   m_Children: []
   m_Father: {fileID: 4368578466105820060}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &2401108532742869652
 MonoBehaviour:
@@ -62552,10 +62552,10 @@
   m_Children: []
   m_Father: {fileID: 4368578466105820062}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &8006045400165783642
 MonoBehaviour:
@@ -62765,8 +62765,8 @@
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_AnchoredPosition: {x: 4.0820866, y: -5.3630285}
+  m_SizeDelta: {x: 343.71146, y: 136.9635}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &4368578466105820036
 MonoBehaviour:
@@ -62855,7 +62855,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -63112,7 +63112,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &4368578466105820046
 MonoBehaviour:
@@ -63200,7 +63200,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -63290,7 +63290,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &4368578466105820041
 MonoBehaviour:
@@ -63379,7 +63379,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -86008,7 +86008,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &6429054435818101727
 MonoBehaviour:
@@ -86097,7 +86097,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -86187,7 +86187,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &6429054435818101720
 MonoBehaviour:
@@ -86275,7 +86275,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -86531,8 +86531,8 @@
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_AnchoredPosition: {x: 4.0820866, y: -5.3630285}
+  m_SizeDelta: {x: 343.71146, y: 136.9635}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &6429054435818101714
 MonoBehaviour:
@@ -86621,7 +86621,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -86708,10 +86708,10 @@
   m_Children: []
   m_Father: {fileID: 6429054435818101704}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &8134652736066097921
 MonoBehaviour:
@@ -86919,10 +86919,10 @@
   m_Children: []
   m_Father: {fileID: 6429054435818101706}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &1044331665746140930
 MonoBehaviour:
@@ -87131,10 +87131,10 @@
   m_Children: []
   m_Father: {fileID: 6429054435818101710}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -7.2701845, y: 10.145175}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &6485177588155483833
 MonoBehaviour:
@@ -103339,7 +103339,7 @@
   m_AnchorMin: {x: 0.5, y: 1}
   m_AnchorMax: {x: 0.5, y: 1}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 679.8618, y: 406.48245}
+  m_SizeDelta: {x: 708.5547, y: 420.55038}
   m_Pivot: {x: 0.5, y: 1}
 --- !u!114 &7091046974789393544
 MonoBehaviour:
@@ -103578,7 +103578,7 @@
   m_AnchorMin: {x: 0, y: 0}
   m_AnchorMax: {x: 1, y: 1}
   m_AnchoredPosition: {x: 0, y: -49.017002}
-  m_SizeDelta: {x: -46.227417, y: -152.23166}
+  m_SizeDelta: {x: -15.94049, y: -152.23166}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &7091046974789393548
 MonoBehaviour:
--- a/Restoration/12_UnityReskin_20260922/Assets/Scenes/RecoveredMain.unity
+++ b/Restoration/12_UnityReskin_20260922/Assets/Scenes/RecoveredMain.unity
@@ -39681,7 +39681,7 @@
   m_AnchorMin: {x: 0.5, y: 1}
   m_AnchorMax: {x: 0.5, y: 1}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 679.8618, y: 406.48245}
+  m_SizeDelta: {x: 708.5547, y: 420.55038}
   m_Pivot: {x: 0.5, y: 1}
 --- !u!1 &148115204847419826
 GameObject:
@@ -39756,8 +39756,8 @@
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0, y: 0}
   m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: -49.017002}
-  m_SizeDelta: {x: -46.227417, y: -152.23166}
+  m_AnchoredPosition: {x: 0, y: -49.017}
+  m_SizeDelta: {x: -15.9404745, y: -152.23166}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &148115204847419830
 GameObject:
@@ -61465,7 +61465,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -61495,7 +61495,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -61533,7 +61533,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -61628,10 +61628,10 @@
   m_Children: []
   m_Father: {fileID: 956705005179947504}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &956705005179947497
 GameObject:
@@ -61853,10 +61853,10 @@
   m_Children: []
   m_Father: {fileID: 956705005179947506}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &956705005179947501
 GameObject:
@@ -61893,10 +61893,10 @@
   m_Children: []
   m_Father: {fileID: 956705005179947510}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -7.2701845, y: 10.145175}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &956705005179947503
 GameObject:
@@ -61937,7 +61937,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &956705005179947505
 GameObject:
@@ -61977,7 +61977,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &956705005179947507
 GameObject:
@@ -62055,8 +62055,8 @@
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_AnchoredPosition: {x: 4.0820866, y: -5.3630285}
+  m_SizeDelta: {x: 343.71146, y: 136.9635}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &956705005179947511
 GameObject:
@@ -88422,8 +88422,8 @@
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_AnchoredPosition: {x: 4.0820866, y: -5.3630285}
+  m_SizeDelta: {x: 343.71146, y: 136.9635}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &2392379486693195491
 GameObject:
@@ -88463,7 +88463,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &2392379486693195493
 GameObject:
@@ -88503,7 +88503,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &2392379486693195495
 GameObject:
@@ -88547,7 +88547,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -88706,7 +88706,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -88736,7 +88736,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -88766,10 +88766,10 @@
   m_Children: []
   m_Father: {fileID: 2392379486693195494}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &2392379486693195513
 GameObject:
@@ -88806,10 +88806,10 @@
   m_Children: []
   m_Father: {fileID: 2392379486693195490}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -7.2701845, y: 10.145175}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &2392379486693195515
 GameObject:
@@ -88846,10 +88846,10 @@
   m_Children: []
   m_Father: {fileID: 2392379486693195492}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &2392379486693195517
 GameObject:
@@ -95817,10 +95817,10 @@
   m_Children: []
   m_Father: {fileID: 3680849620535074616}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -7.2701845, y: 10.145175}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &3680849620535074593
 GameObject:
@@ -95857,10 +95857,10 @@
   m_Children: []
   m_Father: {fileID: 3680849620535074620}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &3680849620535074595
 GameObject:
@@ -96082,10 +96082,10 @@
   m_Children: []
   m_Father: {fileID: 3680849620535074622}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &3680849620535074599
 GameObject:
@@ -96195,7 +96195,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -96233,7 +96233,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -96327,7 +96327,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -96352,8 +96352,8 @@
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_AnchoredPosition: {x: 4.0820866, y: -5.3630285}
+  m_SizeDelta: {x: 343.71146, y: 136.9635}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &3680849620535074617
 GameObject:
@@ -96432,7 +96432,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &3680849620535074621
 GameObject:
@@ -96472,7 +96472,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &3680849620535074623
 GameObject:
@@ -101225,8 +101225,8 @@
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_AnchoredPosition: {x: 4.0820866, y: -5.3630285}
+  m_SizeDelta: {x: 343.71146, y: 136.9635}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &4714375801652915242
 GameObject:
@@ -101305,7 +101305,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &4714375801652915246
 GameObject:
@@ -101345,7 +101345,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &4714375801652915248
 GameObject:
@@ -101382,10 +101382,10 @@
   m_Children: []
   m_Father: {fileID: 4714375801652915241}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -7.2701845, y: 10.145175}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &4714375801652915250
 GameObject:
@@ -101422,10 +101422,10 @@
   m_Children: []
   m_Father: {fileID: 4714375801652915245}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &4714375801652915252
 MonoBehaviour:
@@ -101647,10 +101647,10 @@
   m_Children: []
   m_Father: {fileID: 4714375801652915247}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &4714375801652915256
 MonoBehaviour:
@@ -101747,7 +101747,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -101785,7 +101785,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -101815,7 +101815,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -134071,10 +134071,10 @@
   m_Children: []
   m_Father: {fileID: 7221373871140724383}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &7221373871140724354
 GameObject:
@@ -134111,10 +134111,10 @@
   m_Children: []
   m_Father: {fileID: 7221373871140724379}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -7.2701845, y: 10.145175}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &7221373871140724356
 GameObject:
@@ -134151,10 +134151,10 @@
   m_Children: []
   m_Father: {fileID: 7221373871140724381}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &7221373871140724358
 MonoBehaviour:
@@ -134371,7 +134371,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -134466,7 +134466,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -134560,7 +134560,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -134644,8 +134644,8 @@
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_AnchoredPosition: {x: 4.0820866, y: -5.3630285}
+  m_SizeDelta: {x: 343.71146, y: 136.9635}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &7221373871140724380
 GameObject:
@@ -134685,7 +134685,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &7221373871140724382
 GameObject:
@@ -134725,7 +134725,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!114 &7221373871140724384
 MonoBehaviour:
@@ -135475,10 +135475,10 @@
   m_Children: []
   m_Father: {fileID: 7896409519968824062}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &7896409519968824033
 GameObject:
@@ -135515,10 +135515,10 @@
   m_Children: []
   m_Father: {fileID: 7896409519968824058}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -7.2701845, y: 10.145175}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &7896409519968824035
 GameObject:
@@ -135555,10 +135555,10 @@
   m_Children: []
   m_Father: {fileID: 7896409519968824060}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
-  m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 1, y: 1}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: -64, y: -12}
+  m_AnchorMin: {x: 0.5, y: 0.5}
+  m_AnchorMax: {x: 0.5, y: 0.5}
+  m_AnchoredPosition: {x: -3.1880977, y: 4.7821465}
+  m_SizeDelta: {x: 291.71094, y: 87.67268}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &7896409519968824037
 GameObject:
@@ -135788,7 +135788,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -135947,7 +135947,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -135977,7 +135977,7 @@
       m_Calls: []
   m_Sprite: {fileID: 0}
   m_Type: 0
-  m_PreserveAspect: 1
+  m_PreserveAspect: 0
   m_FillCenter: 1
   m_FillMethod: 0
   m_FillAmount: 1
@@ -136049,8 +136049,8 @@
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_AnchoredPosition: {x: 4.0820866, y: -5.3630285}
+  m_SizeDelta: {x: 343.71146, y: 136.9635}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &7896409519968824059
 GameObject:
@@ -136090,7 +136090,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &7896409519968824061
 GameObject:
@@ -136130,7 +136130,7 @@
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
   m_AnchoredPosition: {x: 0, y: 0}
-  m_SizeDelta: {x: 335.54727, y: 132.30606}
+  m_SizeDelta: {x: 335.54727, y: 126.23745}
   m_Pivot: {x: 0.5, y: 0.5}
 --- !u!1 &7896409519968824063
 GameObject:
--- a/Restoration/12_UnityReskin_20260922/Assets/Resources/CoinReskin/Amounts.png.meta
+++ b/Restoration/12_UnityReskin_20260922/Assets/Resources/CoinReskin/Amounts.png.meta
@@ -49,7 +49,7 @@
   alignment: 0
   spritePivot: {x: 0.5, y: 0.5}
   spritePixelsToUnits: 32
-  spriteBorder: {x: 80, y: 70, z: 80, w: 150}
+  spriteBorder: {x: 90, y: 76, z: 90, w: 198}
   spriteGenerateFallbackPhysicsShape: 1
   alphaUsage: 1
   alphaIsTransparency: 1
@@ -93,6 +93,19 @@
     ignorePlatformSupport: 0
     androidETC2FallbackOverride: 0
     forceMaximumCompressionQuality_BC6H_BC7: 0
+  - serializedVersion: 3
+    buildTarget: Android
+    maxTextureSize: 2048
+    resizeAlgorithm: 0
+    textureFormat: -1
+    textureCompression: 0
+    compressionQuality: 50
+    crunchedCompression: 0
+    allowsAlphaSplitting: 0
+    overridden: 0
+    ignorePlatformSupport: 0
+    androidETC2FallbackOverride: 0
+    forceMaximumCompressionQuality_BC6H_BC7: 0
   spriteSheet:
     serializedVersion: 2
     sprites: []
@@ -102,13 +115,13 @@
     spriteID: 5e97eb03825dee720800000000000000
     internalID: 0
     vertices: []
-    indices:
+    indices: 
     edges: []
     weights: []
     secondaryTextures: []
     nameFileIdTable: {}
-  mipmapLimitGroupName:
+  mipmapLimitGroupName: 
   pSDRemoveMatte: 0
-  userData:
-  assetBundleName:
-  assetBundleVariant:
+  userData: 
+  assetBundleName: 
+  assetBundleVariant: 
